docs/docsite/rst/dev_guide/developing_modules_best_practices.rst

```diff
@@ -32,6 +32,7 @@ General guidelines & tips
 =========================
 
 * Each module should be self-contained in one file, so it can be be auto-transferred by Ansible.
+* Module name should be using underscores instead of hyphens or spaces as word separator. Using hyphens and spaces will prevent Ansible from importing your module.
```

Comment:
```suggestion
* Module name MUST use underscores instead of hyphens or spaces as word separator. Using hyphens and spaces will prevent Ansible from importing your module.
```